package.json

```diff
@@ -50,7 +50,7 @@
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-simple-import-sort": "^4.0.0",
     "fs-extra": "^8.1.0",
-    "mocha": "^7.1.1",
+    "mocha": "^6.1.4",
```

Comment:
Why are we lowering the `mocha` version? Please add a description or revert it back to `^7.1.1`.